WcaOnRails/Gemfile.lock

```diff
@@ -733,4 +738,4 @@ DEPENDENCIES
   wicked_pdf
 
 BUNDLED WITH
-   2.1.4
+   2.2.20
```

Comment:
You seem to have the wrong (i.e. "too new") `bundler` version installed.
Please refer to our project's `README` to figure out how to install the exact bundler version you need and then reinstall the dependencies

Reply:
Nevermind, just realised that the Bundler version isn't included in our README.
Install the appropriate version like so: gem install bundler -v WHATEVER_VERSION

Reply:
Got it, I changed my bundle version to 2.1.4 (which is the version shown in `Gemfile.lock` in thewca repo), deleted `Gemfile.lock` and did `bundle install` again, but a pre-commit hook is preventing me from making commits. Is it expecting bundler with version 2.2.21?
![image](https://user-images.githubusercontent.com/58278719/123551601-1c576f80-d7a5-11eb-9985-efc5babd0937.png)

I looked over the code it listed but I'm not sure where to change what it should expect :/

Reply:
You should be able to run the commit hooks with (almost) any Bundler version and you certainly shouldn't be restricted to 2.2.21. But I'm not a Bundler expert so I don't know how to fix this from the top of my head...

Reply:
Try removing all other Bundler versions except 2.1.4. If that fails, remove Bundler entirely and only re-install 2.1.4.
Bundler is handled like any other Ruby `gem` so googling for "ruby remove gem" or similar should do the trick,